Makefile

```diff
@@ -38,6 +38,9 @@ manifests:
 	STATE_CONTROLLER_IMAGE=$(STATE_CONTROLLER_IMAGE) \
 		hack/generate-manifests.sh
 
+check:
+	./hack/verify-fmt.sh
```

Comment:
maybe we can run `go fmt` and `go vet` then check with git if any file was change and fail the test?

Reply:
should be done now, could you take another look please?